crates/query-engine/translation/src/translation/helpers.rs

```diff
@@ -137,6 +137,21 @@ impl<'a> Env<'a> {
             .ok_or(Error::ProcedureNotFound(procedure_name.to_string()))
     }
 
+    pub fn lookup_generated_mutation(
+        &self,
+        procedure_name: &str,
+    ) -> Result<crate::translation::mutation::generate::Mutation, Error> {
+        // this means we generate them on every mutation request
```

Comment:
When I say "we generate them" I don't mean all the SQL for each, but we work out which mutations we CAN have based on the metadata.

Reply:
This is perfectly fine for a first version.